pkg/converter/target/externalentity.go

```diff
@@ -27,8 +27,7 @@ import (
 
 const (
 	// LabelSizeLimit K8s label requirements, https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/.
-	LabelSizeLimit  = 64
-	LabelExpression = "[^a-zA-Z0-9_-]+"
+	LabelSizeLimit = 63
```

Comment:
Should this be moved to util/k8s/tags.go?

Reply:
yes